sw/host/opentitantool/config/opentitan_cw310.json

```diff
@@ -0,0 +1,26 @@
+{
+  "includes": ["opentitan.json"],
+  "interface": "cw310",
+  "pins": [
+    {
+      "name": "RESET",
+      "mode": "OpenDrain",
```

Comment:
Does it make sense to use `#[serde(rename = "snake_case")]` on this enum so that it's `open_drain` and consistent with everything else, instead?

Reply:
This JSON format is mostly a proof of concept, we have not settled on the final serialization format.  Chris suggested that it was more important to settle on the schema/structure of the data to be part of the configuration, in the form of its Rust struct declarations.
I do not want to spend to much time making the file format nice, until we have more experience with the use of configuration files.

Reply:
I also don't necessarily agree that the values should be coerced into snake_case.  Leaving as StudlyCaps means its a proper noun, meaning someone looking in the code knows to look for a struct or enum name.

Wait until you see what serde_json does with enum variants with substructure :)  (a case where Rust's data model exceeds json's data model).